Shopping.Cart.Tests/ChangeInventoryTranslationTests.cs

```diff
@@ -1,18 +1,17 @@
-using Shopping.Cart.EventStore;
 using Shopping.Cart.Slices;
 
 namespace Shopping.Cart.Tests;
 
 public class ChangeInventoryTranslationTests
 {
     [Test]
-    public async Task Test()
+    public void Test()
     {
         ChangeInventoryCommandHandler changeInventoryCommandHandler = new ChangeInventoryCommandHandler();
         var uncommittedEvent = changeInventoryCommandHandler.Handle(new InventoryChangedExternal(new Guid("00000000-0000-0000-0000-000000000001"), 1));
 
-        Assert.That(uncommittedEvent[0], Is.TypeOf<InventoryChanged>());
-        Assert.That((uncommittedEvent[0] as InventoryChanged).ProductId, Is.EqualTo(new Guid("00000000-0000-0000-0000-000000000001")));
-        Assert.That((uncommittedEvent[0] as InventoryChanged).Inventory, Is.EqualTo(1));
+        Assert.That(uncommittedEvent, Is.TypeOf<InventoryChanged>());
+        Assert.That((uncommittedEvent as InventoryChanged)!.ProductId, Is.EqualTo(new Guid("00000000-0000-0000-0000-000000000001")));
+        Assert.That((uncommittedEvent as InventoryChanged)!.Inventory, Is.EqualTo(1));
```

Comment:
Since `ChangeInventoryCommandHandler.Handle` returns one event the indexer on what I assume _used_ to be an array is invalid syntax.  This change clears that up and the test still passes.